src/Symfony/Component/Translation/TranslatorFallbackInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * TranslatorFallbackInterface.
+ *
+ * @author Bozhidar Hristov <warxcell@gmail.com>
+ */
+interface TranslatorFallbackInterface extends TranslatorInterface
+{
+    /**
+     * Gets the fallback locales.
+     *
+     * @return array $locales The fallback locales
+     */
+    public function getFallbackLocales();
+
+    /**
+     * Sets the fallback locales.
```

Comment:
the method name is self explanatory.

Reply:
I used description as-is from Translator.php

Reply:
You can remove this IMHO.

Reply:
Everywhere else there is a description. 
```php
 /**
     * Returns the current locale.
     *
     * @return string The locale
     */
    public function getLocale();
```

Reply:
I understand the point but the fact is this should be cleaned : https://github.com/symfony/symfony/pull/24931